extensions/micrometer-opentelemetry/deployment/src/main/java/io/quarkus/micrometer/opentelemetry/deployment/MicrometerOtelBridgeProcessor.java

```diff
@@ -0,0 +1,81 @@
+package io.quarkus.micrometer.opentelemetry.deployment;
+
+import java.util.Locale;
+import java.util.function.BooleanSupplier;
+
+import jakarta.enterprise.inject.Instance;
+import jakarta.inject.Singleton;
+
+import org.jboss.jandex.ClassType;
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.ParameterizedType;
+import org.jboss.jandex.Type;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.opentelemetry.api.OpenTelemetry;
+import io.quarkus.arc.deployment.SyntheticBeanBuildItem;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.BuildSteps;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.SystemPropertyBuildItem;
+import io.quarkus.micrometer.deployment.MicrometerProcessor;
+import io.quarkus.micrometer.opentelemetry.runtime.MicrometerOtelBridgeRecorder;
+import io.quarkus.opentelemetry.deployment.OpenTelemetryEnabled;
+import io.quarkus.opentelemetry.runtime.config.build.OTelBuildConfig;
+import io.quarkus.opentelemetry.runtime.config.runtime.OTelRuntimeConfig;
+
+@BuildSteps(onlyIf = {
+        MicrometerProcessor.MicrometerEnabled.class,
+        OpenTelemetryEnabled.class,
+        MicrometerOtelBridgeProcessor.OtlpMetricsExporterEnabled.class })
+public class MicrometerOtelBridgeProcessor {
+
+    @BuildStep
+    @Record(ExecutionTime.RUNTIME_INIT)
+    void createBridgeBean(OTelRuntimeConfig otelRuntimeConfig,
+            MicrometerOtelBridgeRecorder recorder,
+            BuildProducer<SyntheticBeanBuildItem> syntheticBeanProducer,
+            BuildProducer<SystemPropertyBuildItem> systemProperty) {
+
+//        // Avoid users from receiving:
+//        // io.ope.ins.mic.v1_.OpenTelemetryMeterRegistry] (main) A MeterFilter is being configured ...
+//        // registered to this registry...
+//        // It's unavoidable because of how Quarkus startup works and users cannot do anything about it.
+//        // see: https://github.com/micrometer-metrics/micrometer/issues/4920#issuecomment-2298348202
+//        systemProperty.produce(
```

Comment:
@radcortez I wonder why this didn't work here... I had to set these properties in the runtime `application.properties`